content/applications/sales/crm/optimize/lead_enrichment.rst

```diff
@@ -0,0 +1,113 @@
+
+To set up lead enrichment in the *CRM* application, navigate to :menuselection:`CRM app -->
+Configuration --> Settings`. Under the :guilabel:`Lead Generation` section, tick the box next to
+:guilabel:`Lead Enrichment`, and select either :guilabel:`Enrich leads on demand only` or
+:guilabel:`Enrich all leads automatically`. :guilabel:`Save` the changes.
+
+.. image:: lead_enrichment/lead-enrichment-activate.png
+   :align: center
+   :alt: CRM lead generation settings page, with lead enrichment activation highlighted, and enrich
+         leads on demand only chosen.
+
+Enrich leads
+============
+
+Enrichment of leads is based on the domain of the email address of the customer set on the lead.
+There are two different ways that a lead can be enriched: *automatically* and *manually*.
+
+Automatically enrich leads
+--------------------------
+
+During set up, if :guilabel:`Enrich all leads automatically` was selected, then no action needs to
+be taken by the user to enrich the lead. A scheduled action runs automatically, every sixty minutes,
+and enrichment occurs on leads, after a remote database is contacted.
+
+.. tip::
+   To access the cron that runs for the automatic lead enrichment, activate :ref:`developer mode
+   <developer-mode>`, and navigate to :menuselection:`Settings app --> Technical menu --> Automation
+   --> Scheduled Actions`. In the :guilabel:`Search...` type in `partner autocomplete`. Click into
+   the result labeled :guilabel:`Partner Autocomplete Sync with remote DB`, and make any necessary
+   adjustments. In the :guilabel:`Execute Every` field, enter a value greater than or equal to five
+   minutes.
+
+.. example::
+   The following is an example of a successful autocomplete of lead enrichment data:
+
+   .. image:: lead_enrichment/lead-enrichment-data.png
+      :align: center
+      :alt: Chatter showing lead enrichment data.
+
+Manually enrich leads
+---------------------
+
+If the :guilabel:`Enrich leads on demand only` option was selected when activating :guilabel:`Lead
+Enrichment`, each lead that a user wishes to enrich will need to be manually enriched. This is
+achieved by clicking the :guilabel:`Enrich` button in the top menu of the lead.
+
+The same information will be retrieved at the same :abbr:`IAP (In-App Puchase)` credit cost (one per
+enrichment). This method of enrichment is useful when every lead does not need to be enriched or
+cost is an issue.
+
+.. image:: lead_enrichment/manual-enrichment.png
+   :align: center
+   :alt: Manual enrich button feature highlighted on the CRM lead.
+
+.. tip::
+   Manually enrich leads in bulk using the *list* view. First navigate to the user's *CRM* pipeline
+   (:menuselection: CRM app --> Sales --> My Pipeline`) and click the list view (:guilabel:`☰ (three
+   horizontal lines)` icon). Next, tick the checkboxes for the leads that should be manually
+   enriched. Finally, click the :guilabel:`⚙️ Actions` icon, and select :guilabel:`Enrich`.
```

Comment:
While this is technically correct, you are instructing the reader to go to the main pipeline where leads *and* opportunities are...so, selecting and enriching them in this fashion from the 'My Pipeline' page *does* work, but its not specific to leads. So, that said, you may want to consider adding a mention about navigating to the Leads page first, and doing this process. Then, end the blurb/admonition by saying the same process can be achieved on the 'My Pipeline' page with leads *and* opportunities. That's just a suggestion, though -- what do you think?